scripts/mech_events.py

```diff
@@ -25,82 +25,92 @@
 import time
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Any, Dict
+from string import Template
+from typing import Any, ClassVar, Dict
 
 import requests
-from eth_utils import to_checksum_address
 from tqdm import tqdm
-from web3 import HTTPProvider, Web3
 from web3.datastructures import AttributeDict
-from web3.types import BlockParams
 
 
 SCRIPT_PATH = Path(__file__).resolve().parent
 STORE_PATH = Path(SCRIPT_PATH, "..", ".trader_runner")
 MECH_EVENTS_JSON_PATH = Path(STORE_PATH, "mech_events.json")
-AGENT_MECH_JSON_PATH = Path(SCRIPT_PATH, "..", "contracts", "AgentMech.json")
 HTTP = "http://"
 HTTPS = HTTP[:4] + "s" + HTTP[4:]
 CID_PREFIX = "f01701220"
 IPFS_ADDRESS = f"{HTTPS}gateway.autonolas.tech/ipfs/"
-LATEST_BLOCK_NAME: BlockParams = "latest"
-BLOCK_DATA_NUMBER = "number"
-BLOCKS_CHUNK_SIZE = 5000
-EXCLUDED_BLOCKS_THRESHOLD = 2 * BLOCKS_CHUNK_SIZE
-NUM_EXCLUDED_BLOCKS = 10
-MECH_EVENTS_DB_VERSION = 2
+MECH_EVENTS_DB_VERSION = 3
```

Comment:
@DavidMinarsch The easiest way of implementing the subgraph integration is by using a new db. 
This will require to recreate the local DB (a few minutes the first time is run). 
Fortunately, after some tests, it seems this approach is way more robust than querying the blockchain directly in the following sense:
- It has not been observed that the process is interrupted by RPC issues as before
- Even if it takes some time, it is successful in the tests performed.
- The user can decide to interrupt the mech update, and it will have stored progress locally (so it will retake from where it was left, next time the script is run)

Reply:
Yes let's do a new db version